proto/klyraprotocol/blocktime/genesis.proto

```
@@ -4,9 +4,6 @@ package klyraprotocol.blocktime;
 option go_package = "github.com/StreamFinance-Protocol/stream-chain/protocol/x/blocktime/types";
 
 import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)